app/src/main/java/com/odysee/app/callable/CommentEnabled.java

```diff
@@ -0,0 +1,46 @@
+package com.odysee.app.callable;
+
+import android.util.Log;
+
+import com.odysee.app.exceptions.LbryRequestException;
+import com.odysee.app.exceptions.LbryResponseException;
+import com.odysee.app.utils.Lbry;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+public class CommentEnabled implements Callable<Boolean> {
+    private static final String TAG = CommentEnabled.class.getSimpleName();
+    private static final String COMMENT_URL = "https://comments.lbry.com/api/v2";
+    private static final String METHOD_COMMENT_LIST = "comment.List";
+
+    private final String channelId;
+    private final String channelName;
+
+    public CommentEnabled(String channelId, String channelName) {
+        this.channelId = channelId;
+        this.channelName = channelName;
+    }
+
+    @Override
+    public Boolean call() {
+        Map<String, Object> params = new HashMap<>(4);
```

Comment:
Setting an initial capacity isn't really necessary, but if you choose to do so, there are only 3 entries here instead of 4.